### PR TITLE
[WIP] :lipstick: new homepage

### PR DESCRIPTION
**Pull request by AlissonSteffens**
This PR introduces a new homepage for CASSIE

screenshotfull

### CODE REVIEW

File: src/app/components/homepage/Footer.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+export default function Footer(managers, techs) {
+  const classes = useStyles();
+  const [t] = useTranslation()
+  return (
+    <Box bgcolor='black'>
+        <Container className={classes.content} maxWidth="md" display='flex' flexDirection='column'>
+          <Grid container  spacing={1} justify="center">
+            <Grid item xs={12} md={3}>
+              <Typography variant="h6" color="secondary">
+                CASSIE
```
Comment (vinigmartins) [inline src/app/components/homepage/Footer.js:56]: I prefer without "cassie", but I understand the use for identity

File: src/i18n/en.js [hunk 2]
```diff
@@ -1,10 +1,69 @@
...
+                linkVideo: 'https://www.youtube.com/watch?v=7q-3xNVq0tY&'
+            },
+            baysqueeze: {
+                title: 'BAYSQUEEZE',
+                text: 'The project seeks to contribute with actions and results for the preservation of our planet, by assuming the task of fulfilling the United Nations Sustainable Development Goals, as well as attending the national needs of knowledge about climate change, in parallel with the goals of the sub-network "Coastal Zones", of the Brazilian Research Network on Global Climate Change (Rede Clima). ',
+                btn: 'Know more about the project'
+            },
+            riscport: {
+                title: 'RISCPORT',
+                text: 'O projeto procura contribuir com ações e resultados para o bem e preservação do nosso planeta, tomando para si, a tarefa de cumprir os objetivos de desenvolvimento sustentável, estabelecidos pela ONU, bem como atender às necessidades nacionais de conhecimento sobre mudanças climáticas, paralelamente aos objetivos da sub-rede "Zonas Costeiras", da Rede Brasileira de Pesquisas sobre Mudanças Climáticas Globais(Rede Clima).',
```
Comment (vinigmartins) [inline src/i18n/en.js:29]: Change to:

The main objective of this project is to conduct a climate risk analysis (environmental and economic) and to determine possible adaptation measures, based on the future projection of environmental threats (e.g. storm waves), to increase the resilience of the port sector and adjacent areas in the Babitonga Bay, in an interdisciplinary-multiinstitutional network context.

**Review by vinigmartins — CHANGES_REQUESTED** [review]
The new home is marvelous, but needs some changes. 😄

**Comment by AlissonSteffens** [comment]
@ViniciusAS pode validar novamente?

File: src/i18n/pt.js [hunk 3]
```diff
@@ -52,18 +54,59 @@ const pt = {
...
             }
-        },        
+        },
+        problems: {
+            title: "Problemas Conhecidos",
+            text: 'O CASSIE ainda está em desenvolvimento, alguns processos podem falhar. Esses são alguns problemas conhecidos.',
+            '0': {
+                title: 'Analise de muitos dados',
```
Comment (vinigmartins) [inline src/i18n/pt.js:106]: typo corrections:

Análise de muitos dados

**Review by vinigmartins — CHANGES_REQUESTED** [review]
When I switch to English and go to FAQ or KNOWN ISSUES, the site switches back to Portuguese, and vice versa.

File: src/app/components/homepage/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  const [t] = useTranslation();
+  return (
+    <div className={classes.root}>
+      <AppBar position="fixed" color="default">
+        <Toolbar>
+          <Link color="inherit" href="/" className={classes.title}>
+            <Typography variant="h6" color="default">
+              {t('self.title')}
+            </Typography>
+          </Link>
```
Comment (vinigmartins) [inline src/app/components/homepage/NavBar.js:29]: The link is taking up almost the entire header.

image

**Review by vinigmartins — CHANGES_REQUESTED** [review]

**Comment by AlissonSteffens** [comment]
Fixed almost everything. The problem with language changing though, I'll only solve after merging #45

**Review by IsraelEfraim — APPROVED** [review]
New homepage is very appreciable.

**Review by vinigmartins — APPROVED** [review]
Nice 💖

**Pull request merged**